python/ray/dag/compiled_dag_node.py

```diff
@@ -37,6 +37,14 @@
 
 MAX_BUFFER_COUNT = MAX_BUFFER_TOTAL_MEMORY // MAX_BUFFER_SIZE
 
+
+class RayDAGArgs:
+    """Holds the input arguments for an accelerated DAG node."""
+
+    args: any
+    kwargs: any
```

Comment:
```suggestion
    args: Tuple[Any, ...]
    kwargs: Dict[str, Any]
```

Reply:
I think this is no longer necessary as we are now using a namedtuple.

Reply:
Oh I think there is more type hint-friendly syntax for this in recent versions of Python. Here's an example: https://stackoverflow.com/questions/34269772/type-hints-in-namedtuple

Reply:
Done